src/autocomplete.ts

```diff
@@ -74,3 +57,29 @@ function renderAutocomplete(users: IUser[]) {
 
   autocompleteListEl.innerHTML = usersListHtml.join();
 }
+
+function initUserRender() {
+  of(1)
```

Comment:
why do we need of(..) ?

I guess would be better to do like so: 

ajax.getJSON(usersApiUrl).pipe(
    pluck('results')
).subsribe(...